package.json

```diff
@@ -26,7 +26,8 @@
   "resolutions": {
     "glob-parent": "^6.0.1",
     "hosted-git-info": "^3.0.8",
-    "url-parse": "^1.5.3"
+    "url-parse": "^1.5.3",
+    "axios": "^0.21.4"
```

Comment:
Shouldn't we be using `>=` for all these instead of `^` to prevent this from ever holding back an upgrade?

Reply:
I think we want a caret here so that it will check for the latest compatible version but that doesn't pull in any breaking changes. If we did`>=` it might install breaking changes at some point down the road, and it might be hard to pinpoint what had caused it.

Reply:
Okay.  It feels backwards to me, but it's fine, just more micromanagement.